docs/resources/xray_license_policy.md

```diff
@@ -0,0 +1,144 @@
+# Xray License Policy Resource
```

Comment:
I think alex figured out how to generate docs automatically

Reply:
Yes. See https://github.com/jfrog/terraform-provider-project/pull/6

Reply:
Auto-generated documentation added